client/js/models/archive.js

```diff
@@ -79,7 +80,6 @@ module.exports = {
   effects: {
     new: function (data, state, send, done) {
       drive = getDrive()
-      var _fs = {}
```

Comment:
The variable is closed-over and reside in the file callback instance.  if you make it top level rather than  a new lexical in new(), it'll be shared across all uses though.